hiring.qmd

```diff
@@ -288,64 +289,13 @@ Another example:
 >
 > \[interviewer names\]
 
-#### 4. Initial in-person interview
-
-> Dear \[full applicant name\],
->
-> Thank you for interviewing with us recently.
-> We would like to bring you on campus for an in-person interview.
->
-> The interview would be approximately four hours in the Biosciences Research Laboratory building on the University of Arizona campus followed by either lunch or dinner with the group.
-> During the interview, you would be meeting with Dr. Kristina Riemer, who runs the group, the members of his group, and our collaborators in the Data Science Institute.
->
-> We will cover the cost of meals and travel expenses.
-> Please let us know about any accommodations that would facilitate your interview; we want to provide an inclusive and comfortable experience.
->
-> We would like to have the interview on \[date(s)\].
-> You can opt for morning (8am-noon) or afternoon (1-5pm).
-> Please let us know which days and parts of days you prefer.
->
-> We can also do a later date if that would work better, please let us know.
->
-> Thank you for your interest in this position, and we look forward to meeting you.
->
-> Sincerely,
->
-> \[interviewer names\]
-
-#### 5. Details in-person interview
-
-> Dear \[full interviewee name\],
->
-> This is to confirm that you are scheduled to interview with Dr. Kristina Riemer's group on \[date\] from \[time\] at the Biosciences Research Laboratory building on the University of Arizona campus.
->
-> Below is the planned schedule for the interview:
->
-> \[interview schedule\]
->
-> The interview will happen in the Bioscience Research Laboratories (BSRL) building located at 1230 N Cherry Ave (https://goo.gl/maps/yTbKipGD6p2ZvFzb6).
-> Meetings will be on the second floor, which requires a key card to access, so a member of the group will meet you on the south side of the lobby of the building at \[time\].
->
-> If you are driving to campus, you can park in the Highland Parking Garage one block west of the BSRL building (https://goo.gl/maps/HE3eBnQbyUMs2x8Y8).
-> Parking is \$8 per day, which will be covered.
->
-> Another way to get to campus is the streetcar, and the eastern terminus is right next to the BSRL building.
-> Alternatively we will reimburse rideshare service charges.
->
-> Please let us know if you have any questions about the interview format or logistics.
-> We are looking forward to meeting you!
->
-> Sincerely,
->
-> \[interviewer names\]
-
-#### 5b. Full-day Zoom interview
+#### 4. Full-day Zoom interview
```

Comment:
Do we want to keep "5. Details in-person interview" in case we need that section? Maybe someone is local?